### PR TITLE
New gbh logo

### PR DESCRIPTION
**Pull request by brandonhundt**
- Swaps out WGBH logo for the GBH logo

## To Review
- [ ] Use the Preview link located in the Now comment below.

>  ...or...

- [ ] Checkout Branch.
- [ ] Run `yarn`.
- [ ] Run `now dev`.
- [ ] Go to [localhost:3000](https://localhost:3000).

>  ...then...

- [ ] Scroll down to the footer to ensure the new GBH logo is in place.

### CODE REVIEW

File: config/local.json [hunk 1]
```diff
@@ -1 +0,0 @@
-{}
```
Comment (rpeterman-gp) [inline config/local.json:1]: Why was this file modified?

Comment (brandonhundt) [reply config/local.json:1]: I was attempting to get my local dev working. I'll revert the change.
